components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AbstractAPIManager.java

```diff
@@ -1225,6 +1226,35 @@ public String getOpenAPIDefinition(String apiId, String tenantDomain) throws API
         return definition;
     }
 
+    @Override
+    public String getAsyncAPIDefinition(Identifier apiId) throws APIManagementException {
```

Comment:
This method is not needed. lets use the getAsyncAPIDefinition(String apiId, String tenantDomain) method only

Reply:
Removed this method from APIManager interface. Fixed